src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -150,8 +150,7 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
 
     # Check for faulty pre-release helm versions
     if "3.3.0-rc" in helm_version:
-        telemetry.set_user_fault()
-        raise CLIError("The current helm version is not supported for azure-arc onboarding. Please upgrade helm to a stable version and try again.")
+        raise ClientRequestError("The current helm version is not supported for azure-arc onboarding.", recommendation="Please upgrade helm to a stable version and try again.")
```

Comment:
Should we keep this as ValidationError instead of ClientRequestError? ClientRequestError seems to be for client side errors when trying to reach the service(connect RP)